src/mono/netcore/Makefile

```diff
@@ -71,17 +61,3 @@ run-tests-coreclr-all:
 # for `System.Runtime.Tests` (the rule builds both the lib and the test if needed)
 run-tests-corefx-%:
 	$(DOTNET) msbuild /t:RunLibrariesTest /p:LibraryToTest=$* $(MONO_PROJ)
```

Comment:
Out of curiosity, does this work now with your changes?

Reply:
Should work, the targets to patch testhost with mono bits are unchanged.
I'll check.

Reply:
If the testhost patching is unchanged it's probably still broken, just wanted to check.
Relevant issue: https://github.com/dotnet/runtime/issues/40853

Reply:
ah, same output here.

Reply:
Fixed, should work now